src/main/scala/nl/knaw/dans/easy/fedora2vault/DDM.scala

```diff
@@ -0,0 +1,259 @@
+       { emd.getEmdSource.getDcSource.asScala.map(bs => <dc:source xml:lang={ lang(bs) }>{ bs.getValue }</dc:source>) }
+       { emd.getEmdType.getDcType.asScala.map(bs => <dct:type xsi:type={ dcType(bs) }>{ bs.getValue }</dct:type>) }
+       { emd.getEmdFormat.getDcFormat.asScala.map(bs => <dct:format>{ bs.getValue }</dct:format>) }
+       { emd.getEmdFormat.getTermsExtent.asScala.map(notImplemented("extent format")) }
+       { emd.getEmdFormat.getTermsMedium.asScala.map(notImplemented("medium formt")) }
+       { emd.getEmdSubject.getDcSubject.asScala.filter(isAbr).map(bs => <dc:subject xml:lang={ lang(bs) } xsi:type={ abrType(bs) }>{ bs.getValue }</dc:subject>) }
+       { emd.getEmdSubject.getDcSubject.asScala.filterNot(isAbr).map(bs => <ddm:subject xml:lang={ lang(bs) } subjectScheme={ bs.getScheme }>{ bs.getValue }</ddm:subject>) }
+       { emd.getEmdCoverage.getDcCoverage.asScala.map(bs => <dct:coverage xml:lang={ lang(bs) }>{ bs.getValue }</dct:coverage>) }
+       { emd.getEmdCoverage.getTermsSpatial.asScala.filter(isAbr).map(bs => <dct:spatial xml:lang={ lang(bs) } xsi:type={ abrType(bs) }>{ bs.getValue }</dct:spatial>) }
+       { emd.getEmdCoverage.getTermsSpatial.asScala.filterNot(isAbr).map(bs => <dct:spatial xml:lang={ lang(bs) } xsi:type={ xsiType(bs) }>{ bs.getValue }</dct:spatial>) }
+       { emd.getEmdCoverage.getTermsTemporal.asScala.filter(isAbr).map(bs => <dct:temporal xml:lang={ lang(bs) } xsi:type={ abrType(bs) }>{ bs.getValue }</dct:temporal>) }
+       { emd.getEmdCoverage.getTermsTemporal.asScala.filterNot(isAbr).map(bs => <dct:temporal xml:lang={ lang(bs) } xsi:type={ xsiType(bs) }>{ bs.getValue }</dct:temporal>) }
+       { emd.getEmdCoverage.getEasSpatial.asScala.filterNot(_.getPlace == null).map(notImplemented("places")) }
+       { dateMap.filter(isOtherDate).map { case (key, values) => values.map(_.withLabel(dateLabel(key))) } }
+       { emd.getEmdCoverage.getEasSpatial.asScala.filterNot(_.getBox == null).map(notImplemented("boxes")) }
+       { emd.getEmdCoverage.getEasSpatial.asScala.filterNot(_.getPoint == null).map(notImplemented("points")) }
+       { emd.getEmdCoverage.getEasSpatial.asScala.filterNot(_.getPolygons == null).map(notImplemented("polygons")) }
+       <dct:license xsi:type="dct:URI">{ toUri(emd.getEmdRights) }</dct:license>
+       { emd.getEmdLanguage.getDcLanguage.asScala.map(bs => <dct:language xsi:type={langType(bs)}>{ langValue(bs) }</dct:language>) }
+     </ddm:dcmiMetadata>
+   </ddm:DDM>
+ }
+
+  private def langType(bs: BasicString) = bs.getSchemeId match {
+    case "fra" |"fra/fre" | "deu" | "deu/ger" | "nld" | "nld/dut" | "eng" => "dct:ISO639-3"
+    case _ => null
+  }
+
+  private def langValue(bs: BasicString) = bs.getValue match {
+    case "fra/fre" => "fra"
+    case "deu/ger" => "deu"
+    case "nld/dut" => "nld"
+    case "eng" => "eng"
+    case s => s
+  }
+
+  private def isAbr(string: BasicString) = string.getScheme == "ABR"
```

Comment:
From looking at https://github.com/DANS-KNAW/easy-ddm/blob/dc1f7107a48350e2ecb023d775a03ea7054ce9d8/src/main/java/nl/knaw/dans/pf/language/ddm/handlermaps/NameSpace.java#L30-L40, I wonder if this shouldn't be

```suggestion
  private def isAbr(string: BasicString) = string.getScheme == "abr"
```

but perhaps you should do a case insensitive match if you found `"ABR"` somewhere else?